ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -216,6 +241,24 @@ private MessageWriteSetStateInStore checkWriteSetStateInStore(MessageWriteSet me
     }
   }
 
+  private void checkCapacityAndUpdateHelix(long totalCapacity, long usedCapacity) {
```

Comment:
i think it's useful here to document the function and describe _what_ is updated.

Reply:
yeah, it's complicated enough that I think it warrants a comment